web/gds-user-ui/src/modules/dashboard/certificate/lib/trixoQuestionnaireValidationSchema.ts

```diff
@@ -10,13 +10,14 @@ export const trixoQuestionnaireValidationSchema = yup.object().shape({
         regulator_name: yup.string()
       })
     ),
-    financial_transfers_permitted: yup.string().oneOf(['no', 'yes', 'partial']).default('no'),
-    has_required_regulatory_program: yup.string().oneOf(['no', 'yes', 'partial']).default('no'),
+    financial_transfers_permitted: yup.string().oneOf(['no', 'yes', 'partially']).default('no'),
+    has_required_regulatory_program: yup.string().oneOf(['no', 'yes', 'partially']).default('no'),
```

Comment:
This update is not doing anything since we don't use it but i make the updates for future